gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -37,7 +37,7 @@ public StorageStubProvider(
 
   public StorageBlockingStub buildBlockingStub() {
     return StorageGrpc.newBlockingStub(
-        ComputeEngineChannelBuilder.forAddress("storage.googleapis.com", 443)
+        ComputeEngineChannelBuilder.forAddress(readOptions.getGrpcHostname(), 443)
```

Comment:
There are `ComputeEngineChannelBuilder.forTarget(String)` method that seems to allow override not only a hostname but scheme and port too. Can we sue it instead to have a greater flexibility?

Reply:
It's a wrapper of .forTarget method, which utilizes handy GrpcUtil.authorityFromHostAndPort method to build target string.
.forTarget doesn't gain extra flexibility for this use case.